plasmapy_nei/nei/nei.py

```diff
@@ -1,7 +1,1359 @@
 """Contains classes to represent non-equilibrium ionization simulations."""
 
-__all__ = ["NEI"]
+__all__ = ["NEI", "NEIError", "SimulationResults"]
 
 
-class NEI:
+import numpy as np
+from typing import Union, Optional, List, Dict, Callable
+import astropy.units as u
+import plasmapy as pl
+from scipy import interpolate, optimize
+from plasmapy_nei.eigen import EigenData
+from plasmapy.atomic import IonizationStates
```

Comment:
The attached PR wraps this in a try/except. Why do you use this ancient version, anyway? :p